src/test/indexTest.ts

```diff
@@ -3,6 +3,7 @@ const expect = chai.expect;
 const assert = chai.assert;
 
 import childProcess = require('child_process');
+import { doesNotThrow } from 'assert';
```

Comment:
Is this required?

Reply:
@martysweet Not sure how that got in there.  I'll remove it.